requirements.txt

```diff
@@ -6,3 +6,4 @@ passlib==1.7.1
 pymongo==3.6.1
 pyzmq==17.0.0
 Werkzeug==0.14.1
+Twisted==18.9.0
```

Comment:
The code does not require ```Twisted``` and it runs pefrectly fine without it.
It's just that one way to deploy the application is to use ```Twisted``` but it's not a requirement.

Perhaps readme could be a bit clearer in this case.

Reply:
Right. Missed this.